content/zh-cn/docs/concepts/services-networking/service.md

```diff
@@ -816,6 +818,8 @@ Kubernetes `ServiceTypes` 允许指定你所需要的 Service 类型。
 -->
 * `ClusterIP`：通过集群的内部 IP 暴露服务，选择该值时服务只能够在集群内部访问。
   这也是你没有为服务显式指定 `type` 时使用的默认值。
+  你可以使用向公众暴露服务通过使用[Ingress](/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1/)或者
+  [Gateway API](https://gateway-api.sigs.k8s.io/)。
```

Comment:
```suggestion
  你可以使用 [Ingress](/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1/)
  或者 [Gateway API](https://gateway-api.sigs.k8s.io/) 向公众暴露服务。
```

Reply:
Sorry, I will be careful next time. Thanks for the correction.